code/datums/components/crafting/tailoring.dm

```diff
@@ -50,6 +50,24 @@
 	time = 20
 	category = CAT_CLOTHING
 
+/datum/crafting_recipe/hudsunciv
+	name = "Civilian HUDsunglasses"
+	result = /obj/item/clothing/glasses/hud/civ/sunglasses
+	time = 20
```

Comment:
use SECONDS defines

Reply:
I'm not quite sure what you mean by this. I more or less just copy/pasted the security HUDglasses recipe with the names/items changed.

Reply:
we have defines for time. use 2 SECONDS instead of 20